### PR TITLE
feat: add chain spec for rococo

### PR DESCRIPTION
**Pull request by green-jay**
https://app.clickup.com/t/2jzhueq
CU-2jzhueq

### CODE REVIEW

**Comment by lumir-mrkva** [comment]
Task linked: [CU-2jzhueq HydraDX Parachain deployment on Rococo](https://app.clickup.com/t/2jzhueq)

File: node/src/chain_spec/rococo.rs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+		),
+		// Protocol ID
+		Some(PROTOCOL_ID),
+		// Fork ID
+		None,
+		// Properties
+		Some(properties),
+		// Extensions
+		Extensions {
+			relay_chain: "polkadot".into(),
```
Comment (lumir-mrkva) [inline node/src/chain_spec/rococo.rs:125]: rococo

File: node/src/command.rs [hunk 2]
```diff
@@ -47,6 +47,7 @@ fn load_spec(id: &str, is_testing: bool) -> std::result::Result<Box<dyn sc_servi
 			"" => Box::new(chain_spec::hydradx::parachain_config()?),
 			"local" => Box::new(chain_spec::local::parachain_config()?),
 			"staging" => Box::new(chain_spec::staging::parachain_config()?),
+			"rococo" => Box::new(chain_spec::rococo::parachain_config()?),
```
Comment (lumir-mrkva) [inline node/src/command.rs:50]: it should be loaded from pregenerated file, not generated every node run

**Review by lumir-mrkva — CHANGES_REQUESTED** [review]

File: node/src/chain_spec/rococo.rs [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+					hex!["2cb1a0ef4ce819893905e3a6a8e46b652c43aee6c154921220902cabfdcfdd07"].into(),
+					1_200_000_000 * UNITS,
+				)],
+				// parachain ID
+				PARA_ID.into(),
+			)
+		},
+		// Bootnodes
+		vec![
+			"/dns/p2p-01.hydra.hydradx.io/tcp/30333/p2p/12D3KooWCtBQpwnWV7yMaEyBRkcAcAej78Q2uZawk5RcDrYktVQS"
```
Comment (devtomnl) [inline node/src/chain_spec/rococo.rs:99]: Could we maybe avoid using the legacy hydradx.io domain ?

Comment (green-jay) [reply node/src/chain_spec/rococo.rs:99]: what is the bootnode url then?

File: node/src/command.rs [hunk 4]
```diff
@@ -105,6 +105,7 @@ impl SubstrateCli for Cli {
 				"hydradx" => Box::new(chain_spec::hydradx::parachain_config()?),
 				"local" => Box::new(chain_spec::local::parachain_config()?),
 				"staging" => Box::new(chain_spec::staging::parachain_config()?),
+				"rococo" => Box::new(chain_spec::rococo::parachain_config()?),
```
Comment (lumir-mrkva) [inline node/src/command.rs:108]: you have added the JSON, but it has to be loaded as well here

**Review by lumir-mrkva — APPROVED** [review]

**Pull request merged**